configs/common/packages.yaml

```diff
@@ -70,7 +70,7 @@
       version: [1.1.0]
     fms:
       version: [2023.01]
-      variants: precision=32,64 +quad_precision +gfs_phys +openmp +pic constants=GFS
+      variants: precision=32,64 +quad_precision +gfs_phys +openmp +pic constants=GFS build_type=Release
```

Comment:
@AlexanderRichert-NOAA Is this an unintentional leftover from debugging for the UFS?

Reply:
i believe this was actually required to solve issues for certain UFS RTs that use debugging flags and were failing with an FMS version w/out this variant.